java/runtime/src/main/java/org/ray/runtime/context/LocalModeWorkerContext.java

```diff
@@ -24,7 +24,7 @@ public LocalModeWorkerContext(JobId jobId) {
 
   @Override
   public UniqueId getCurrentWorkerId() {
-    throw new NotImplementedException();
+    throw new NotImplementedException("getCurrentWorkerId() not (yet) implemented.");
```

Comment:
Suggest to change the reason to `Not supported in SINGLE_PROCESS mode.` as this is not a public API and it’s only referenced in `RayNativeRuntime`, which means we won’t implement it for single process mode.